src/standalone/activation/activationManager.ts

```diff
@@ -28,7 +29,15 @@ export class ExtensionActivationManager implements IExtensionActivationManager {
         this.syncActivationServices.map((item) => item.activate());
     }
     public async activate(): Promise<void> {
-        // Activate all activation services together.
-        await Promise.all([this.singleActivationServices.map((item) => item.activate())]);
+        // Activate all activation services together. Don't fail them all if one fails.
+        await Promise.all([
+            this.singleActivationServices.map(async (item) => {
+                const promise = item.activate();
```

Comment:
I had a bug in my activate and it made the extension not start. Figured this was better.